test/cases/enum.ts

```diff
@@ -6,7 +6,8 @@ export var schema = {
       "enum": ["a", "b", "c"]
     },
     "bar": {
-      "enum": [1, 2, 3]
+      "enum": [1, 2, 3],
+      "tsEnumNames": ["One","Two","Three"]
```

Comment:
Thanks for the background. I think I'm missing something.

If you have the schema

```json
{
  type: "integer",
  enum: [1, 2, 3],
  tsEnumNames: ["Voice", "Chat", "Email"]
}
```

Which compiles to 

```ts
export enum MediaType {
    Voice = 1,
    Chat = 2,
    Email = 3
}
```

You can then consume it as

```ts
const myMediaType = MediaType.Chat
```

Which will compile to the JS

```js
const myMediaType = 2
```

Is this what you need? If so, (1) let's only support number enum members, and (2) what are the runtime helpers for?